CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#776](https://github.com/MyIntervals/emogrifier/pull/776))
 
 ### Changed
+- `CssConcatenator::$mediaRules` was given dedicated classes
```

Comment:
I'm a big fan of using the active voice. Please let's change this to: "Give …"

Reply:
https://github.com/joelparkerhenderson/git_commit_message

Reply:
will take a look tomorrow after I see how step 1 of #807 goes (brain sleepy)